docs/premium-features/break-the-monolith.md

```diff
@@ -6,61 +6,64 @@ slug: /break-the-monolith
 
 # Break The Monolith
 
-Amplication's new "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. Using AI, Amplication simplifies the transition of your entities to different services, streamlining your shift to a microservices architecture.
+Amplication's "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. It introduces a new drag-and-drop interface to simplify the process of moving entities across different services.
 
-This guide will teach you everything so you can best utilize this feature for your apps.
+Additionally, you have the option to utilize our new AI Helper for further guidance and automation in this transition, optimizing the process of moving to a microservices architecture.
+
+This guide will show you how to effectively use this feature for your services.
 
 :::note
-The "Break The Monolith" feature is available exclusively to Enterprise workspaces, with trial workspaces having limited access.
+The "Break The Monolith" feature is **available exclusively to Enterprise workspaces including trial workspaces**.
```

Comment:
BTW- Free users will get access to the overview mode, without the ability to "redesign" the architecture. Not sure we need to mention it, but mostly as FYI